modules/radsBidAdapter.js

```diff
@@ -125,10 +162,25 @@ function isVideoRequest(bid) {
   return bid.mediaType === 'video' || !!utils.deepAccess(bid, 'mediaTypes.video');
 }
 
-function prepareExtraParams(params, payload) {
+function prepareExtraParams(params, payload, bidderRequest) {
   if (params.pfilter !== undefined) {
     payload.pfilter = params.pfilter;
   }
+
+  if (bidderRequest && bidderRequest.gdprConsent) {
+    if (payload.pfilter !== undefined) {
+      if (!payload.pfilter.gdpr_consent) {
+        payload.pfilter.gdpr_consent = bidderRequest.gdprConsent.consentString;
+        payload.pfilter.gdpr = bidderRequest.gdprConsent.gdprApplies;
+      }
+    } else {
+      payload.pfilter = {
+        'gdpr_consent': bidderRequest.gdprConsent.consentString,
+        'gdpr': bidderRequest.gdprConsent.gdprApplies
+      };
+    }
```

Comment:
Shouldn't this initialize as a new object with the bidderRequest.gdprConsent values every time prepareExtraParams runs?

Reply:
this is always initialized, except if an `gdpr_consent` already exists in `pfilter` or an empty `bidderRequest.gdprConsent`

Reply:
I have removed this feature from the code.  It was needed for tests